openmetadata-ui/src/main/resources/ui/src/components/TeamDetails/UserTab/UserTab.component.tsx

```diff
@@ -0,0 +1,264 @@
+    return (
+      <ErrorPlaceHolder
+        button={
+          <UserSelectableList
+            hasPermission
+            selectedUsers={currentTeam.users ?? []}
+            onUpdate={onAddUser}>
+            <Button
+              ghost
+              className="p-x-lg"
+              data-testid="add-new-user"
+              icon={<PlusOutlined />}
+              title={
+                permission.EditAll
+                  ? t('label.add-new-entity', { entity: t('label.user') })
+                  : t('message.no-permission-for-action')
+              }
+              type="primary">
+              {t('label.add')}
+            </Button>
+          </UserSelectableList>
+        }
+        className="mt-0-important"
+        heading={t('label.user')}
+        permission={permission.EditAll}
+        type={ERROR_PLACEHOLDER_TYPE.ASSIGN}
+      />
+    );
+  }
+
+  return (
+    <Row gutter={[16, 16]}>
+      <Col span={24}>
+        <Row justify="space-between">
+          <Col span={8}>
+            <Searchbar
+              removeMargin
+              placeholder={t('label.search-for-type', {
+                type: t('label.user-lowercase'),
+              })}
+              searchValue={searchText}
+              typingInterval={500}
+              onSearch={onSearchUsers}
+            />
+          </Col>
+          <Col>
+            <Space>
+              {users.length > 0 && permission.EditAll && (
+                <UserSelectableList
+                  hasPermission
+                  selectedUsers={currentTeam.users ?? []}
+                  onUpdate={onAddUser}>
+                  <Button data-testid="add-new-user" type="primary">
+                    {t('label.add-entity', { entity: t('label.user') })}
+                  </Button>
+                </UserSelectableList>
+              )}
+              <ManageButton
+                canDelete={false}
+                entityName={currentTeam.name}
+                extraDropdownContent={IMPORT_EXPORT_MENU_ITEM}
+              />
+            </Space>
+          </Col>
+        </Row>
+      </Col>
+
+      {isLoading > 0 ? (
+        <Loader />
+      ) : (
+        <Col span={24}>
+          <Table
+            bordered
+            className="teams-list-table"
+            columns={columns}
+            dataSource={sortedUser}
+            locale={{
+              emptyText: <FilterTablePlaceHolder />,
+            }}
+            pagination={false}
+            rowKey="name"
+            size="small"
+          />
+          {paging.total > PAGE_SIZE_MEDIUM && (
+            <NextPrevious
+              currentPage={currentPage}
+              isNumberBased={Boolean(searchText)}
+              pageSize={PAGE_SIZE_MEDIUM}
+              paging={paging}
+              pagingHandler={onChangePaging}
+              totalCount={paging.total}
+            />
+          )}
+        </Col>
+      )}
+      <Modal
+        centered
+        destroyOnClose
```

Comment:
I feel we have confirmation modal, can you check that?

Reply:
We have marked it as deprecated. we are going to remove it in the future!